server/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <title>Tocco</title>
   <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" name="viewport"/>
```

Comment:
This will only affect the local dev server. If important on not already done, this has to be done on nice side as well.